app/src/test/java/com/codesoom/assignment/domain/UserTest.java

```diff
@@ -0,0 +1,34 @@
+package com.codesoom.assignment.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class UserTest {
+    @Nested
+    @DisplayName("User 클래스는")
```

Comment:
`@Test`를 잘 읽어보면 이 테스트코드가 테스트하고 있는 대상은 `User.builder`라는 것을 알 수 있습니다.
```suggestion
    @DisplayName("User.builder는")
```

Reply:
user에 바꿔주면서 같이 바꿔줘야 했군요 생각도 못하고 있었습니다